src/site/content/en/reliable/resilient-search-experiences/index.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: Resilient search experiences
+authors:
+  - demianrenzulli
+  - jeffposnick
+date: 2018-06-16
```

Comment:
Is this an old doc?

Reply:
Sorry, I copied the header from another post, and forgot to edit the date. Correcting this now.